e2e/integration_tests/backends_test.ts

```diff
@@ -27,27 +28,28 @@ import {SMOKE} from './constants';
 /**
  *  This file tests backend switching scenario.
  */
-
+// TODO: Support backend switching between wasm and cpu.
+// https://github.com/tensorflow/tfjs/issues/7623
 describeWithFlags(
     `${SMOKE} backend switching`, {
-      predicate: testEnv => testEnv.backendName === 'webgl' &&
-          tfc.findBackend('webgl') !== null && tfc.findBackend('cpu') !== null
+      predicate: testEnv =>
+          testEnv.backendName !== 'cpu' && testEnv.backendName !== 'wasm'
     },
 
-    () => {
-      it(`from webgl to cpu.`, async () => {
-        await tfc.setBackend('webgl');
+    (env) => {
+      it(`from ${env.name} to cpu.`, async () => {
+        await tfc.setBackend(env.name);
 
-        const webglBefore = tfc.engine().backend.numDataIds();
+        const backendBefore = tfc.engine().backend.numDataIds();
 
         const input = tfc.tensor2d([1, 1, 1, 1], [2, 2], 'float32');
-        // input is stored in webgl backend.
+        // input is stored in webgl or webgpu backend.
```

Comment:
```suggestion
        // input is stored in backend.
```

Reply:
Please don't assume we only have WebGL and WebGPU backends in all the comments below.